poker-pals/src/components/Login.vue

```diff
@@ -104,12 +104,20 @@
       });
 
       // Navigate to tables page if authentication was successful
-      this.socket.on("authenticatedUser", userID => {
-        this.$router.replace({ name: "Tables", params: {authenticated: true, socket: this.socket, userID: userID}});
-      });
+      this.socket.on("authenticated", userData => {
+        // if the user is banned, navigate to banned page
+        this.socket.on("banned", () => {
+          this.$router.replace({ name: "Banned", params: {authenticated: true}});
+        });
+        
+        // navigate to reports page if the user is an admin
+        if(userData.isAdmin) {
+          this.$router.push({ name: "Reports", params: {authenticated: true, socket: this.socket, userData: userData}});
+          return;
+        }
 
-      this.socket.on("banned", () => {
-        this.$router.replace({ name: "Banned", params: {authenticated: true}});
```

Comment:
is this code removed for a reason? the server is still sending the message

Reply:
I moved it to the top

Reply:
oh sorry I didn't see that, guess I'm blind